Makefile

```diff
@@ -293,9 +303,6 @@ create-management-cluster: $(KUSTOMIZE) $(ENVSUBST) $(KUBECTL) $(KIND) ## Create
 	# Install cert manager and wait for availability
 	./hack/install-cert-manager.sh
 
-	# Create secret for AzureClusterIdentity
-	./hack/create-identity-secret.sh
```

Comment:
Do any of the upstream tests rely on this? Are they already set up for workload identity?

Reply:
I would expect so (though not certain), and that we would update the tests as we move their CAPZ dependencies forward to the release when these changes land.